recipes/dustmaps/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dustmaps" %}
+{% set version = "0.1a12" %}
```

Comment:
Would it be possible to not build from an alpha release?

Reply:
Only if @gregreen releases one, but the package is functional.

Reply:
I've been calling every release up until now "alpha." That's probably being conservative at this point though.

Reply:
Seems fine to me to just push it out as `0.1`, which indicates that it's an unfinished state. Unless you think it's finished, in which case I'd suggest `1.0`...